jbpm-wb-dashboard/jbpm-wb-dashboard-client/src/test/java/org/jbpm/dashboard/renderer/client/TaskDashboardTest.java

```diff
@@ -570,6 +592,22 @@ public void testOpenInstanceDetails() {
         verify(placeManager).goTo(TASK_DETAILS_SCREEN);
     }
 
+    @Test
+    public void testNofificationWhenDetailsNotAvailable() {
```

Comment:
You have a typo in the test name.
Just thinking now whether it would make sense to align test names with the same type of tests in ```ProcessDashboardTest```. Since they are testing the same thing only in different perspective.